spanner/session.go

```diff
@@ -227,6 +227,11 @@ func (s *session) String() string {
 func (s *session) ping() error {
 	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
 	defer cancel()
+
+	// Start parent span that doesn't record.
+	parentCtx, span := StartSpan(ctx, "ping", WithSampler(NeverSample()))
```

Comment:
`parentCtx` is not used anywhere in the rest of this function.

Reply:
Should it be something like: 

```go
	ctx = trace.StartSpan(ctx, "cloud.google.com/go/spanner.BatchUpdate")
	defer func() { trace.EndSpan(ctx, err) }()
```

Reply:
My mistake. I see the purpose of the change is to never sample `ping()`.

Reply:
That's really bizarre. I ran `go test` and it didn't tell me about any of this. I wonder why.

All the build errors are fixed now, sorry about that.

Reply:
May be `go vet` or `golint` will tell you this.

Reply:
You're right, `go vet` would've shown me.